lib/index.js

```diff
@@ -3,9 +3,4 @@
 var Server = require('./server');
 
 
-// Declare internals
```

Comment:
Ok, I think you are confused by that.  The style guide states that you "can only have two top level globals" referring to "internals" and "exports".  It does not say they are required to be there even if they are unused.  "can" vs "must".

And going by the lack of such in other modules that don't use them, I'd say it is fine to remove this one.